widgets/settings/settingsdialog.cpp

```diff
@@ -802,7 +802,11 @@ bool setAutostartEnabled(bool enabled)
         desktopFile.write("[Desktop Entry]\n"
                           "Name=" APP_NAME "\n"
                           "Exec=\"");
```

Comment:
BTW I'm not sure why the `\"` quoting is needed - usually these are not present in `.desktop` files..

Reply:
When I remember correctly, it broke otherwise with spaces in the path.

Reply:
See https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#exec-variables